Cargo.toml

```diff
@@ -23,12 +23,14 @@ include = ["Cargo.toml", "LICENSE", "README.md", "src/**", "tests/**", "examples
 thiserror = "1.0"
 rustc-hash = "1.1.0"
 serde = { version = "1.0", features = ["derive"], optional = true }
+log = { version = "0.4.14", default-features = false } # for debug logs in tests
```

Comment:
No objection, just wondering:
What features are we trying to opted out of? Why `default-features = false`?